builtin/providers/aws/resource_aws_vpc.go

```diff
@@ -260,7 +260,9 @@ func resourceAwsVpcUpdate(d *schema.ResourceData, meta interface{}) error {
 		d.SetPartial("enable_dns_support")
 	}
 
-	if d.HasChange("enable_dns_support") {
+	_, hasEnableDnsSupportOption := d.GetOk("enable_dns_support")
+
+	if !hasEnableDnsSupportOption || d.HasChange("enable_dns_support") {
```

Comment:
seems `enable_dns_support` is the only attribute that defaults `true` on AWS side here, which is why this is the only one that needs this check (if I understand correctly)